README.md

```diff
@@ -27,6 +27,16 @@ When you deploy this website to Azure you can define the [Backend](https://githu
 
 > Note: you can change the InstrumentationKey of the **Application Insight** that is configured by default.
 
+In order to change the backend, set `apiBaseUrl` parameter on the ARM template provided to the url where your aks is configured.
```

Comment:
Can we improve the writing?
If you want to update the application to use your own [backend](https://github.com/Microsoft/TailwindTraders-Backend), set....

Reply:
Changed as you requested